templates/concert-ticket/templates/main.tpl

```diff
@@ -0,0 +1,173 @@
+        </paragraph>
+        {{range .RulesOfAttendance}}
+            <paragraph text-align="left" margin="0 0 0 0" line-height="1.3">
+                <text-chunk font="helvetica" font-size="9">{{.}}</text-chunk>
+            </paragraph>
+        {{end}}
+    </division>
+ </table-cell>
+ <table-cell border-width-top="0.5" border-width-bottom="0.5" border-width-right="0.5" vertical-align="top" indent="0" border-style="single">
+    <division margin="0 0 15 15" padding="0, 0, 0, 0">
+        <paragraph text-align="left" margin="10 0 10 0">
+            <text-chunk font="helvetica-bold" font-size="12">Rules of purchase</text-chunk>
+        </paragraph>
+        {{range .RulesOfPurchase}}
+            <paragraph text-align="left" margin="0 0 0 0" line-height="1.3">
+                <text-chunk font="helvetica" font-size="9">{{.}}</text-chunk>
+            </paragraph>
+        {{end}}
+    </division>
+ </table-cell>
+</table>
+
+<table columns="3" margin="10 0 0 10" column-widths="0.5 0.20 0.30">
+    <table-cell vertical-align="top">
+        <division margin="0 20 0 0" padding="0, 0, 0, 0">
+            <paragraph text-align="top" margin="0 0 0 0">
+                <text-chunk font="helvetica-bold" font-size="12">Program/Bands List/Event Description</text-chunk>
+            </paragraph>
+            <paragraph text-align="top" margin="5 0 0 0" line-height="1.3">
+                <text-chunk font="helvetica" font-size="9">A City literally built on Rock n' Roll in Cadott WI, </text-chunk>
+                <text-chunk font="helvetica" font-size="9"> Rock Fest is the true Rock experience you can't miss. In its 26th year, </text-chunk>
+                <text-chunk font="helvetica" font-size="9"> it is the top venue for people of all ages to come together for one common purpose: </text-chunk>
+                <text-chunk font="helvetica" font-size="9">to congregate with other rock fans from across the world, in a place where rock music still matters.</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Featuring the very best of active and classic rock and legendary names in Rock Music, entertainment and experience are the first priority.</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Aerosmith, Iron Maiden, Avenged Sevenfold, Kiss, Motley Crue, Fleetwood Mac, Tom Petty, Kid Rock, Shinedown,</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Five Finger Death Punch, Rob Zombie, Korn &amp; many more rock legends have graced this permanent Main Stage over the course of the last two and a half decades.</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+    <table-cell vertical-align="top">
+        <image src="path('./templates/res/map.png')" height="160" width="100" margin="5 0 0 0"></image>
+    </table-cell>
+    <table-cell vertical-align="top">
+        <division>
+            <paragraph text-align="top" margin="0 0 0 0">
```

Comment:
`top` is not a valid value for the `text-align` attribute of paragraphs.
Also, the default value for the `margin` attribute is `0`. No need to specify it in this case.